packages/addon-dev/src/rollup-addon-dependencies.ts

```diff
@@ -33,7 +35,27 @@ export default function emberExternals(): Plugin {
   return {
     name: 'ember-externals',
 
-    async resolveId(source) {
+    async resolveId(source, importer) {
+      let isRelative = source.startsWith('.') || source.startsWith('/');
+
+      /**
+        * examples:
+        *
+        * ./button
+        * ../button
+        *
+        */
+      if (isRelative && importer && !path.extname(source)) {
+        let dir = path.dirname(importer);
+        let target = path.resolve(dir, source);
+
+        let result = glob.sync(`${target}.*`);
```

Comment:
Sounds as expensive fs work :(

Reply:
yeah, idk how to resolve template-only imports otherwise :sweat_smile:

Reply:
But there's only one possible legal file extension for a template-only component. `.hbs`. You don't need a wildcard.